.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -54,6 +63,7 @@ jobs:
             -DIREE_ENABLE_LLD=ON \
             -DIREE_ENABLE_ASSERTIONS=ON \
             -DIREE_BUILD_DOCS=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
             -DIREE_TARGET_BACKEND_WEBGPU_SPIRV=ON
       - name: CMake - build
```

Comment:
Can you add a `ccache --show-stats` call after the build step, then push a few commits? I'd expect the first run to populate the cache and subsequent runs to use it.

Something like this:

```diff
- name: CMake - build
        run: |
+         ccache --zero-stats
+         ccache --show-stats
          cmake --build ${BUILD_DIR} -- -k 0
          cmake --build ${BUILD_DIR} --target install -- -k 0
          cmake --build ${BUILD_DIR} --target iree-test-deps -- -k 0
+         ccache --show-stats
```